kodi_addon_checker/schema_validation.py

```diff
@@ -115,6 +115,17 @@ def _validate(xml, schemapath):
 
 
 def check_version(branch_name, schema_file):
+    """Check whether the schema files for the specific
+       branch exists or not
```

Comment:
not fully correct, as it searches for the highest available variant of a schema file